ajax/libs/react-trend/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "react-trend",
+  "filename": "src/index.min.js",
+  "description": "react-trend React component",
+  "homepage": "https://unsplash.github.io/react-trend/",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/unsplash/react-trend.git"
+  },
+  "author": "Joshua Comeau <josh@unsplash.com>",
+  "keywords": [
+    "react-component",
+    "react",
+    "trend",
+    "svg",
+    "graph",
+    "data-visualization"
+  ],
+  "npmName": "react-trend",
+  "npmFileMap": [
+    {
+      "basePath": "src",
+      "files": [
+        "**/*"
```

Comment:
A little hint for this library, we need all files under `basePath` which you will find out soon, so the pattern here is correct 👍